l10n_nl_tax_statement/models/l10n_nl_vat_statement.py

```diff
@@ -72,10 +72,8 @@ def default_get(self, fields_list):
     @api.onchange('date_range_id')
     def onchange_date_range_id(self):
         if self.date_range_id and self.state == 'draft':
-            self.write({
-                'from_date': self.date_range_id.date_start,
-                'to_date': self.date_range_id.date_end,
-            })
+            self.from_date = self.date_range_id.date_start
+            self.to_date = self.date_range_id.date_end
```

Comment:
this is just fine too, but are you aware you can use the [`.update`](https://github.com/OCA/OCB/blob/10.0/odoo/models.py#L5015) function for exactly those cases? A lot more convenient for bigger amounts of fields